### PR TITLE
[feat] Snackbar

### PR DESCRIPTION
**Pull request by Ronak-Mutha**
# Snackbar
- Baseline
image

- Leading
image

- Stacked
image

### CODE REVIEW

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  position: fixed;
+  bottom: 10%;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.leading {
+  position: fixed;
+  bottom: 10%;
+  left: 0%;
```
Comment (saadahmadhere) [inline components/snackbar.css:62]: left 0 would be fine too. No need to give percentage here.

**Review by Prabhav-Sharma — APPROVED** [review]

File: docs/snackbar.html [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                        <li class="m-2">
+                            <a href="../index.html" class="br-2">Home</a>
+                        </li>
+                        <li class="m-2 active">
+                            <a href="/docs/installation.html" class="br-2">Docs</a>
+                        </li>
+                        <li class="m-2">
+                            <a href="https://github.com/Ronak-Mutha/unstoppable-ui" target="_blank"
+                                rel="noopener noreferrer" class="br-2">Contribute <i
+                                    class="fab fa-github-square"></i></a>
```
Comment (saadahmadhere) [inline docs/snackbar.html:43]: </a> could be on the next line for consistency.

**Review by saadahmadhere — APPROVED** [review]
Beautiful and clean. Well structured code. Good to merge.

**Comment by Ronak-Mutha** [comment]
Got 2 reviews. Merging to dev and closing this branch.

**Pull request merged**